web/forms.py

```
@@ -1787,3 +1792,97 @@ class StudyGroupForm(forms.ModelForm):
+        features = self.cleaned_data.get("features")
+        if features:
+            try:
+                if isinstance(features, str):
+                    import json
+
+                    features = json.loads(features)
+
+                if not isinstance(features, list):
+                    raise forms.ValidationError("Features must be a list of items")
+            except json.JSONDecodeError:
+                raise forms.ValidationError("Features must be in valid JSON format")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)